src/preferences/qml/MuseScore/Preferences/videopreferencesmodel.cpp

```
@@ -0,0 +1,79 @@
+}
+
+VideoPreferencesModel::~VideoPreferencesModel()
+{
+    if (videoEncoderResolver()) {
+        disableVideoExportSettingMode();
+    }
+}
+
+void VideoPreferencesModel::load()
+{
+    videoEncoderResolver()->loadedFFmpegChanged().onNotify(this, [this]() {
```

Cppcheck (2.20.0)

[error] There is an unknown macro here somewhere. Configuration is required. If Q_PROPERTY is a macro then please configure it.

(unknownMacro)